INSTALL.asciidoc

```diff
@@ -67,7 +94,8 @@ use of git repo with with openQA
 
 to use os-autoinst with openQA it needs to be "installed". Placing a
 symlink to /usr/lib/os-autoinst that points to the git checkout should
-work. To make tests and needles writable don't touch distri/ and use
+work, as well as symlinking isotovideo, debugviewer and snd2png to /usr/bin.
```

Comment:
No, you shouldn't do this. Either use from local paths or install it with the proper distribution utilities, e.g. `zypper`